charmhub-releaser.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
```

Comment:
just a side comment, I believe we can achieve this same functionality with for example `charmhub-lp-tool -p openstack copy-channel -s zed/edge -d zed/stable`